Source/Windows10/Prism.Windows/PrismApplicationBase.cs

```diff
@@ -233,6 +185,115 @@ protected virtual void RegisterRequiredTypes(IContainerRegistry containerRegistr
             containerRegistry.RegisterSingleton<IModuleInitializer, ModuleInitializer>();
         }
 
+        #region Factory Methods
+
+        private INavigationService CreateNavigationService(Frame frame, CoreWindow window, params Gesture[] gestures)
```

Comment:
should this be protected?

Reply:
my thought is for now no... we can always open it up later